aeron-client/src/main/cpp/ClientConductor.h

```diff
@@ -446,13 +446,14 @@ class CLIENT_EXPORT ClientConductor
 
         if (nowMs > (m_timeOfLastKeepaliveMs + KEEPALIVE_TIMEOUT_MS))
         {
-            if (nowMs > (m_driverProxy.timeOfLastDriverKeepalive() + m_driverTimeoutMs))
+            
+            int64_t lastKeepaliveMs = m_driverProxy.timeOfLastDriverKeepalive();
+            if (nowMs > (lastKeepaliveMs + m_driverTimeoutMs))
             {
                 m_driverActive = false;
 
-                int64_t age = m_driverProxy.timeOfLastDriverKeepalive() + m_driverTimeoutMs;
```

Comment:
Another annoying thing about this was the two separate calls to timeOfLastDriverKeepalive().
In theory it could've changed in between and you end up logging a nonsense "age"